uiowa/uiowa
# Provisions sprint 137

Fixes https://github.com/uiowa/uiowa/issues/5918
# How to test


`blt umc foo.uiowa.edu --simulate --no-db --no-commit` to see whether or not a config directory exists. 

WARNING: local.blt.yml will be re-written during this, so use your local history to revert back to your customized version.

discard changes/additions
